types/protractor-console-plugin/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for protractor-console-plugin 0.1
+// Project: https://github.com/angular/protractor-console-plugin
+// Definitions by: Adam Kwiatek <https://github.com/akwiatek>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface ProtractorConsolePluginConfig {
```

Comment:
Based on https://github.com/angular/protractor-console-plugin/blob/07672cfb635501a6d48dd977eddb2388f34af827/README.md?plain=1#L10

Reply:
IMO this looks like CommonJS module, with exported public interface of the class-like module (old function-constructor syntax being used). There are module level methods (a.k.a. static) exported automatically through instance as well, defined directly on 'ConsolePlugin)
https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require
https://github.com/angular/protractor-console-plugin/blob/07672cfb635501a6d48dd977eddb2388f34af827/index.js#L117